### PR TITLE
Typo in kube-lego README.md

### PR DESCRIPTION
**Pull request by ppwfx**

### CODE REVIEW

**Comment by ppwfx** [comment]
I signed it!

File: stable/kube-lego/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The command removes all the Kubernetes components associated with the chart and
 
 ## Configuration
 
-The following tables lists the configurable parameters of the Prometheus chart and their default values.
+The following tables lists the configurable parameters of the chart and their default values.
```
Comment (mgoodness) [inline stable/kube-lego/README.md]: Let's also make it "table" while we're at it.

**Comment by mgoodness** [comment]
@k8s-bot ok to test

**Comment by mgoodness** [comment]
Made the "tables->table" fix myself. LGTM

**Pull request merged**